Storage/SQL/BrowserDB.swift

```diff
@@ -34,8 +34,9 @@ open class BrowserDB {
 
     // Remove the DB op from the queue (by marking it cancelled), and if it is already running tell sqlite to cancel it.
     // At any point the operation could complete on another thread, so it is held weakly.
-    public func cancel(_ weakDBOperationGetter: () -> Cancellable?) {
-        weak var databaseOperation = weakDBOperationGetter()
+    // Swift compiler bug: failing to compile WeakRef<Cancellable> here.
+    public func cancel(databaseOperation: WeakRef<AnyObject>) {
```

Comment:
@rnewman This is super annoying that AnyObject is needed here, am open to better workaround suggestions.